internal/store/postgres/scheduler/job_run_repository.go

```diff
@@ -232,6 +232,65 @@ func (j *JobRunRepository) GetByScheduledTimes(ctx context.Context, t tenant.Ten
 	return jobRunList, nil
 }
 
+func (j *JobRunRepository) GetP95DurationByJobNames(ctx context.Context, jobNames []scheduler.JobName, lastNRuns int, bufferPercentage int) (map[scheduler.JobName]*time.Duration, error) {
```

Comment:
bufferpercentage can be applied in the estimator during computation this way P95computation can be generic.